### PR TITLE
Add layer name in convmixer

### LINKED ISSUE #1571: Add layer names [ConvMixer]

### PR DESCRIPTION
**Pull request by Anirban0011**
# What does this PR do?

* Attempts to potentially fix  #1571 

#### Special thanks to @IMvision12 for intellectually co-authoring the PR

### CODE REVIEW

**Comment by IMvision12** [comment]
@Anirban0011 for formatting you can use black : `black --line-length 80 file_path`

**Comment by Anirban0011** [comment]
@LukeWood , @ianstenbit , kindly review whenever possible, thanks!

File: keras_cv/models/convmixer.py [hunk 1]
```diff
@@ -106,20 +107,24 @@ def apply_conv_mixer_layer(x, dim, kernel_size):
     Returns:
         the updated input tensor.
     """
+    if name is None:
+        name = str(backend.get_uid("apply_conv_mixer_layer"))
```
Comment (LukeWood) [inline keras_cv/models/convmixer.py:111]: This breaks our pretrained weights, and `apply_` should not end up in layer names.

Comment (Anirban0011) [reply keras_cv/models/convmixer.py]: removed apply_ from names

Comment (Anirban0011) [reply keras_cv/models/convmixer.py]: > This breaks our pretrained weights, and `apply_` should not end up in layer names.

@LukeWood , then name string should not be generated this way if name is `None` ?

**Pull request closed without merging**